src/index.html

```diff
@@ -11,8 +11,72 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="“anonymous”"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <!-- <h1>Product cards</h1> -->
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="./images/imac.jpeg"
+        alt="iMac-icon"
+        class="card__image"
+      />
+
+      <a
+        href="#"
+        class="card__title"
+      >
+        APPLE A1419 iMac 27" Retina
+        <br />
+        5K Monoblock (MNED2UA/A)
+      </a>
```

Comment:
```suggestion
      <h2 class="card__title">
        APPLE A1419 iMac 27" Retina
        <br />
        5K Monoblock (MNED2UA/A)
      </h2>
```

Reply:
<img width="623" alt="image" src="https://github.com/user-attachments/assets/f5c4edb1-c393-4f0f-8277-6da8c258f953" />
it should be the p tag instead of link